{{cookiecutter.module_name}}/main.tf

```diff
@@ -0,0 +1 @@
+
```

Comment:
Would it be useful to include a very simple example? 
```suggestion
# Configure the AWS Provider
provider "aws" {
  version = "~> 2.0"
  region  = "us-east-1"
}

# Create a resource
resource "aws_vpc" "example" {
  cidr_block = "10.0.0.0/16"
}
```

Reply:
It is a good practice to include a provider config in a module, but leave it empty. Provider values should really only be set in the root config, not in the module...

```hcl
provider "aws" {}
```

Of course, this assumes the module is actually based on the aws provider, which is not necessarily true. hmmm... maybe this is being a bit too opinionated and just leaving an empty file is best. users need to do their own research on tf module development...

Reply:
I wouldn't be oppose to created an examples folder with a basic example but think the root level *.tf files should remain empty

Reply:
Could also link to the modules we create in the readme...